centos_ci.sh

```diff
@@ -298,7 +298,7 @@ function build_and_test() {
   cd $GOPATH/src/github.com/minishift/minishift
   make prerelease synopsis_docs link_check_docs
   # Run integration test with 'kvm' driver
-  MINISHIFT_VM_DRIVER=kvm make integration_all
+  MINISHIFT_VM_DRIVER=kvm make integration_pr
```

Comment:
This sound like a plan but the problem is we still don't have a nightly setup for centos CI which means if we merge this PR then other tests will be ignored. I think we should first take care of nightly builds and then pr specific one. @minishift/minishift-dev any other suggestion?

Reply:
@praveenkumar If we merge this it will ignore the coolstore and this feature, nothing will change in coverage. But I agree, that in next sprint it would be good to work on creating nightly.

Reply:
New target for running PR specific tests is the right direction since at some point we need to decide what tests need to be run in PR.

Reply:
It beats me why we cannot create this nightly build. This is nothing else than a new entry in https://github.com/minishift/minishift-ci-jobs/blob/master/minishift-ci-index.yaml with a cron based trigger. @budhrg or am I missing something. Why do we need to wait for another sprint to put this in place?